NOTICE

```diff
@@ -4,8 +4,10 @@ Copyright 2024 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
-This product includes software developed at
-Datastrato (https://datastrato.ai).
+The code is based on Gravitino, that has been kindly donated to the Apache
+Software Foundation by Datastrato (https://datastrato.ai/).
+
+The code was Copyright 2023-2024 Datastrato and/or its affiliates.
```

Comment:
Please remove "and/or its affiliates" as we only had a CLA from Datastrato. It also doesn't make much sense to say based on Gravitino when it is still called Gravitino. I would just say something like this:
"The initial code for the Gravitino project was donated to the ASF by Datastrato (https://datastrato.ai/) copyright 2023-2024."

Reply:
Hi @justinmclean would you please check this (https://github.com/apache/netbeans/blob/master/NOTICE), I referred from Netbeans' NOTICE file.

Reply:
Datastrao is not Oracle, Oracle has legal affilates, Datastaro doesn't. In general  we shouldn't copy from another project unless we know their history and reasons why they did something in that way. The above text I suggested is a better fit given the history of this codebase. The ASF policy here is about putting a relocated copyright line in NOTICE when the headers that were replaced by ASF headers. See https://www.apache.org/legal/src-headers.html#notice (point 2).

Reply:
> Datastrao is not Oracle, Oracle has legal affilates, Datastaro doesn't

As far as I know, Datastrato also has affiliates, and the contributors may from different affiliates, not sure if it is OK to mention?

Reply:
Affiliates above mean in the legal sense not just contributors. As per ASF policy we shouldn't mention anything that was not in the previous Datastrato header.

Reply:
The NetBeans NOTICE file has that wording as their original header said "Copyright 1997-2010 Oracle and/or its affiliates." or similar.

Reply:
I see. Waiting for @JunpingDu 's input.